docs/changelog/1_0_a5.rst

```diff
@@ -0,0 +1,60 @@
+===========
+1.0 Alpha 5
+===========
+
+This changelog summarizes new features and breaking changes in
+EdgeDB 1.0 alpha 5 "Luhman 16".
+
+
+EdgeQL
+======
+
+* Implement casts between JSON and enums, arrays and tuples
+  (:eql:gh:`#251`).
+* Allow constraints on tuple types (:eql:gh:`#1576`).
+* Allow constraints directly on object types in SDL (:eql:gh:`#1164`)
+* Disallow using :eql:constraint:`exclusive` on scalar types
+  (:eql:gh:`#1575`).
+* Proper implementation of ``SET/DROP OWNED``.
+* Fix issues with some ``FOR`` statements (:eql:gh:`#1594`).
+* Use fully-qualified names to disambiguate the expressions produced
+  by :eql:stmt:`DESCRIBE` (:eql:gh:`#1254`).
+* Initial implementation of ``INSERT ... ON CONFLICT ... ELSE``
```

Comment:
I'd expand with a few examples here and there. That would make the changelog more interesting to read. This one for example would benefit from that.

Reply:
I'll link it to the relevant docs section for `INSERT` instead. See #1706